website/content/docs/agent/config/agent-config-files.mdx

```diff
@@ -1,534 +1,11 @@
-  This overrides the default Serf WAN port 8302. This is available in Consul 1.2.2
-  and later.
-
-- `-server` ((#\_server)) - This flag is used to control if an agent is
-  in server or client mode. When provided, an agent will act as a Consul server.
-  Each Consul cluster must have at least one server and ideally no more than 5 per
-  datacenter. All servers participate in the Raft consensus algorithm to ensure that
-  transactions occur in a consistent, linearizable manner. Transactions modify cluster
-  state, which is maintained on all server nodes to ensure availability in the case
-  of node failure. Server nodes also participate in a WAN gossip pool with server
-  nodes in other datacenters. Servers act as gateways to other datacenters and forward
-  traffic as appropriate.
-
-- `-server-port` ((#\_server_port)) - the server RPC port to listen on.
-  This overrides the default server RPC port 8300. This is available in Consul 1.2.2
-  and later.
-
-- `-non-voting-server` ((#\_non_voting_server)) <EnterpriseAlert inline /> - **This field
-  is deprecated in Consul 1.9.1. See the [`-read-replica`](#_read_replica) flag instead.**
-
-- `-read-replica` ((#\_read_replica)) <EnterpriseAlert inline /> - This
-  flag is used to make the server not participate in the Raft quorum, and have it
-  only receive the data replication stream. This can be used to add read scalability
-  to a cluster in cases where a high volume of reads to servers are needed.
-
-- `-syslog` ((#\_syslog)) - This flag enables logging to syslog. This is
-  only supported on Linux and OSX. It will result in an error if provided on Windows.
-
-- `-ui` ((#\_ui)) - Enables the built-in web UI server and the required
-  HTTP routes. This eliminates the need to maintain the Consul web UI files separately
-  from the binary.
-
-- `-ui-dir` ((#\_ui_dir)) - This flag provides the directory containing
-  the Web UI resources for Consul. This will automatically enable the Web UI. The
-  directory must be readable to the agent. Starting with Consul version 0.7.0 and
-  later, the Web UI assets are included in the binary so this flag is no longer necessary;
-  specifying only the `-ui` flag is enough to enable the Web UI. Specifying both
-  the '-ui' and '-ui-dir' flags will result in an error.
-
-<!-- prettier-ignore -->
-- `-ui-content-path` ((#\_ui\_content\_path)) - This flag provides the option
-  to change the path the Consul UI loads from and will be displayed in the browser.
-  By default, the path is `/ui/`, for example `http://localhost:8500/ui/`. Only alphanumerics,
-  `-`, and `_` are allowed in a custom path.`/v1/` is not allowed as it would overwrite
-  the API endpoint.
-
-## Configuration Files ((#configuration_files))
+# Configuration Files ((#configuration_files))
 
 In addition to the command-line options, configuration can be put into
 files. This may be easier in certain situations, for example when Consul is
```

Comment:
```suggestion
You can create one or more files to configure the Consul agent on startup. We recommend grouping similar configurations into separate files, such as ACL parameters, to make it easier to manage configuration changes. Using external files may be easier than configuring agents on the command-line when Consul is
```
Have to spread this suggestion out because of the way the breaks in the lines happen.